python/ray/experimental/channel/torch_tensor_nccl_channel.py

```diff
@@ -50,31 +61,49 @@ def __init__(
         writes the worker-local
         ray.experimental.channel.serialization_context._SerializationContext
         when serializing data.
+
+        Args:
+            writer: The actor that may write to the channel. None signifies the
+                driver.
+            readers: The actors that may read from the channel. None signifies
+                the driver.
```

Comment:
We should sync about this. I changed this part of the API in #45786, so we should discuss how to make them consistent.

Reply:
No worries, I can take care of that here.